api/src/pcapi/core/search/commands/indexation.py

```diff
@@ -136,6 +129,9 @@ def _partially_index(
         page += 1
 
 
+# FIXME (dbaty, 2023-02-17): remove `process_offers_from_database`
+# command when it's been replaced by `partially_index_offers` command
+# in staging rebuild process.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**🎯 suggestion:** ‏Dans le commit message tu dis : "We'll keep them (with their current name) for 2 more versions"
Ça peut être pratique de mettre à partir de quelle version ce `FIXME` est actionable dans ce commentaire, comme ça si il passe à la trappe et qu'on tombe dessus plus tard on saura qu'on peut supprimer ce bout de code

Reply:
Bien vue, très bonne idée, je vais faire ça. Merci !